plugins/catalog-backend/src/ingestion/HigherOrderOperations.ts

```diff
@@ -141,13 +142,40 @@ export class HigherOrderOperations implements HigherOrderOperation {
     );
   }
 
+  // Gets an application token to refresh github locations
+  private async getToken(): Promise<string> {
+    if (process.env.AUTH_GITHUB_TOKEN) {
+      return process.env.AUTH_GITHUB_TOKEN;
+    }
```

Comment:
Can we use the `GITHUB_TOKEN` here that we already have in place?
https://github.com/backstage/backstage/blob/a2b83df986c47d77aca65e9c5aa951e48b558e29/app-config.yaml#L81-L85

Reply:
Replace it with GITHUB_TOKEN...
Thanks!